golang/cosmos/x/lien/keeper/keeper_test.go

```diff
@@ -126,7 +126,7 @@ func makeTestKit() testKit {
 	ms.MountStoreWithDB(bankStoreKey, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(stakingStoreKey, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(lienStoreKey, sdk.StoreTypeIAVL, db)
-	ms.LoadLatestVersion()
+	_ = ms.LoadLatestVersion()
```

Comment:
```
golang/cosmos/x/vbank/vbank_test.go:248:22: Error return value of `ms.LoadLatestVersion` is not checked (errcheck)
	ms.LoadLatestVersion()
```

Reply:
Might as well check that this error return is nil. It's fine to panic.